server/__init__.py

```diff
@@ -140,3 +141,4 @@ def init_flask_restful_routes(app):
     api.add_resource(UserSetLevel,                  '/api/v1/user/<string:username>/set-level/<string:level>')
     api.add_resource(UserAcceptLicense,             '/api/v1/user/accept-license/<int:license_id>')
     api.add_resource(IntersectingTilesAPI,          '/api/v1/grid/intersecting-tiles')
+    api.add_resource(SplitTaskAPI,                  '/api/v1/split/split-task/project/<int:project_id>/task/<int:task_id>')
```

Comment:
To make consistent with other APIs I'd change the shape to

'/api/v1/project/<int:project_id>/task/<int:task_id>/split